### PR TITLE
Release/1.3.1

### PR DESCRIPTION
**Pull request by Pixcell**

### CODE REVIEW

File: docs/md/iofogctl_move.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## iofogctl move
+
+Move an existing resources inside the current ECN
```
Comment (lkrcal) [inline docs/md/iofogctl_move.md]: sorry :-)
`Move existing resources inside the current ECN`

File: docs/md/iofogctl_move.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Synopsis
+
+Move an existing resources inside the current ECN
```
Comment (lkrcal) [inline docs/md/iofogctl_move.md]: same

File: docs/md/iofogctl_move.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+### Examples
+
+```
+iofogctl move microservice NAME AGENT_NAME
```
Comment (lkrcal) [inline docs/md/iofogctl_move.md]: should it explicitly list all the resources it can move?

File: pkg/iofog/install/local_container.go [hunk 4]
```diff
@@ -106,6 +105,15 @@ func NewLocalAgentConfig(name string, image string, ctrlConfig *LocalContainerCo
 	if image == "" {
 		image = "docker.io/iofog/agent:" + util.GetAgentTag()
 	}
+
+	var bindings []string
+
+	if runtime.GOOS == "windows" {
```
Comment (lkrcal) [inline pkg/iofog/install/local_container.go]: should this be in release before 1.4.0?

File: pkg/iofog/install/local_container.go [hunk 5]
```diff
@@ -331,27 +336,27 @@ func (lc *LocalContainer) DeployContainer(containerConfig *LocalContainerConfig)
...
+	// 	networkResponse, err := lc.client.NetworkCreate(ctx, networkName, types.NetworkCreate{
+	// 		Driver:         "bridge",
+	// 		CheckDuplicate: true,
+	// 	})
+	// 	if err != nil {
+	// 		fmt.Printf("Failed to create network: %v\n", err)
+	// 		return "", err
+	// 	}
+	// 	networkID = networkResponse.ID
+	// }
```
Comment (lkrcal) [inline pkg/iofog/install/local_container.go]: intentional comment?

File: docs/md/iofogctl_move.md (hunk 1 above)
Comment (SergeRadinovich) [inline docs/md/iofogctl_move.md]: typo: an existing resources

File: docs/md/iofogctl_move.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+## iofogctl move
```
Comment (SergeRadinovich) [inline docs/md/iofogctl_move.md]: Do we want to patch move command into 1.3?

File: internal/deploy/connector/local.go [hunk 7]
```diff
@@ -63,10 +63,31 @@ func (exe *localExecutor) cleanContainers() {
...
 }
 
+func getController(namespace string) (ctrl config.Controller, err error) {
+	controllers, err := config.GetControllers(namespace)
+	if err != nil {
+		fmt.Print("You must deploy a Controller to a namespace before deploying any Connector")
+		return
+	}
+	if len(controllers) != 1 {
+		return ctrl, util.NewInternalError("Only support 1 controller per namespace")
```
Comment (SergeRadinovich) [inline internal/deploy/connector/local.go]: Only support 1 controller per namespace in a local deployment

File: internal/deploy/connector/local.go [hunk 8]
```diff
@@ -63,10 +63,31 @@ func (exe *localExecutor) cleanContainers() {
...
 	// Deploy connector image
 	util.SpinStart("Deploying Connector")
 
+	controller, err := getController(exe.namespace)
+	if err != nil {
+		return err
+	}
+
+	if util.IsLocalHost(controller.Host) == false {
+		return util.NewInputError("Cannot deploy a local connector with a remote controller")
```
Comment (SergeRadinovich) [inline internal/deploy/connector/local.go]: Connector and Controller should be capitalized as they are proper nouns. Have done this elsewhere in the code and docs

File: pkg/iofog/install/local_container.go (hunk 4 above)
Comment (SergeRadinovich) [inline pkg/iofog/install/local_container.go]: Do we want to patch windows support into 1.3?

File: pkg/iofog/install/local_container.go [hunk 9]
```diff
@@ -331,27 +336,27 @@ func (lc *LocalContainer) DeployContainer(containerConfig *LocalContainerConfig)
...
-			Driver:         "bridge",
-			CheckDuplicate: true,
-		})
-		if err != nil {
-			fmt.Printf("Failed to create network: %v\n", err)
-			return "", err
-		}
-		networkID = networkResponse.ID
-	}
+	// networkName := "local-iofog-network"
```
Comment (SergeRadinovich) [inline pkg/iofog/install/local_container.go]: Delete?

File: pkg/iofog/install/local_container.go [hunk 10]
```diff
@@ -360,11 +365,11 @@ func (lc *LocalContainer) DeployContainer(containerConfig *LocalContainerConfig)
 	}
 
 	// Connect to network
-	err = lc.client.NetworkConnect(ctx, networkID, container.ID, nil)
-	if err != nil {
-		fmt.Printf("Failed to connect container to network: %v\n", err)
-		return "", err
-	}
+	// err = lc.client.NetworkConnect(ctx, networkID, container.ID, nil)
```
Comment (SergeRadinovich) [inline pkg/iofog/install/local_container.go]: Delete?

File: pkg/iofog/install/local_container.go [hunk 11]
```diff
@@ -360,11 +365,11 @@ func (lc *LocalContainer) DeployContainer(containerConfig *LocalContainerConfig)
...
+	// if err != nil {
```
Comment (SergeRadinovich) [inline pkg/iofog/install/local_container.go]: Delete?

File: pkg/iofog/install/local_container.go (hunk 4 above)
Comment (Pixcell) [reply pkg/iofog/install/local_container.go]: As dicussed, comes form a cherry-pick. It is working code, I would leave it there but not market iofogctl as supporting windows yet.

**Pull request merged**